conf/modules.config

```
@@ -52,7 +52,7 @@ if (params.mode == 'AF2') {
 if (params.mode == 'colabfold') {
     process {
         withName: 'RUN_COLABFOLD' {
-            ext.args   = params.useGPU ? '' : '--cpu'
+            ext.args   = [params.useGPU ? '' : '--cpu', params.useAmber ? '--amber' : ''].join(' ').trim()
```

To  make it clear I changed the format, also I think we have to change the name of the parameters for consistency with other nf-core pipelines where the snake_case is used for parameters.
```suggestion
           ext.args = [
                 params.use_gpu ? '' : '--cpu', 
                 params.use_amber ? '--amber' : ''
           ].join(' ').trim()
```